lib/dpul_collections/solr.ex

```diff
@@ -40,7 +40,8 @@ defmodule DpulCollections.Solr do
       fl: fl,
       sort: sort_param(search_state),
       rows: search_state[:per_page],
-      start: pagination_offset(search_state)
+      start: pagination_offset(search_state),
+      uf: "* _query_"
```

Comment:
Is this required to allow the embedded mlt query? Maybe we could add a comment here about why we have this value.